### PR TITLE
Fixes infinite loop in bitrunning

### LINKED ISSUE #79190: using the power of THE NET to mostly-safely humanify ape escape helmet'd monkeys
1. strap an ape escape helmet to a monkey. if the helmet is removed from their head or they change species, they will be ghosted (this is intended functionality), so don't do that yet.
2. put a linked quantum spin inverter inside of a backpack and have the monkey wear it.
3. have the monkey enter THE NET using a bitrunning pod
4. activate the other quantum spin inverter. this will cause the monkey's body to leave the bitrunning pod without ending their bitrunning session.
5. use a genetics console to humanify the monkey. this will trigger the ape escape helmet's recoil effect, but because the monkey's client is not in their body, they will not be ghosted. There is still around a 7.5% chance of an additional negative side effect happening that could harm their body, though (e.g. gibbing or gorillification).
6. have the monkey leave THE NET and reenter their body.
7. they'll return to their body, but be left in this weird state where they can move around and talk, but can't use their hands.
8. have them enter a different bitrunning pod and then disconnect again.
9. the monkey is now a perfectly normal human with access to their hands and such.
10. the first pod you put the monkey in is now in this weird state where it thinks it's occupied but actually isn't. you can't crowbar it open, and deconstructing it will produce a machine frame and parts as normal, but won't destroy it. sadly, deconstructing it again will not produce more machine frames/parts.

I wish I had pictures for this, but they didn't save properly.

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Clears up some functionality in netpod disconnect cases that lead to an infinite loop

Also fixes an errant message from #78959
## Why It's Good For The Game
Fixes #79190

## Changelog
:cl:
fix: Entering a virtual domain should no longer give you a message that it doesn't forbid items
/:cl:

### CODE REVIEW

File: code/modules/bitrunning/objects/netpod.dm [hunk 1]
```diff
@@ -142,10 +142,15 @@
 	open_machine()
 
 /obj/machinery/netpod/open_machine(drop = TRUE, density_to_set = FALSE)
-	unprotect_and_signal()
 	playsound(src, 'sound/machines/tramopen.ogg', 60, TRUE, frequency = 65000)
 	flick("[base_icon_state]_opening", src)
 
+	var/datum/component/netpod_healing/healing_eff = occupant?.GetComponent(/datum/component/netpod_healing)
+	if(healing_eff)
+		qdel(healing_eff)
```
Comment (Mothblocks) [inline code/modules/bitrunning/objects/netpod.dm:150]: `_effect`

Comment (Mothblocks) [reply code/modules/bitrunning/objects/netpod.dm:150]: you also dont need to null check here

File: code/modules/bitrunning/objects/netpod.dm [hunk 2]
```diff
@@ -232,6 +234,23 @@
...
+
+	target.AddComponent(/datum/component/netpod_healing, \
+		brute_heal = 4, \
+		burn_heal = 4, \
+		toxin_heal = 4, \
+		clone_heal = 4, \
+		blood_heal = 4, \
+	)
+
+	target.playsound_local(src, 'sound/effects/submerge.ogg', 20, TRUE)
```
Comment (Mothblocks) [inline code/modules/bitrunning/objects/netpod.dm:250]: vary = TRUE

File: code/modules/bitrunning/objects/netpod.dm [hunk 3]
```diff
@@ -142,10 +142,15 @@
 	open_machine()
 
 /obj/machinery/netpod/open_machine(drop = TRUE, density_to_set = FALSE)
-	unprotect_and_signal()
 	playsound(src, 'sound/machines/tramopen.ogg', 60, TRUE, frequency = 65000)
 	flick("[base_icon_state]_opening", src)
 
+	var/datum/component/netpod_healing/healing_eff = occupant?.GetComponent(/datum/component/netpod_healing)
```
Comment (Mothblocks) [inline code/modules/bitrunning/objects/netpod.dm:148]: this really ought to have been a signal

Comment (jlsnow301) [reply code/modules/bitrunning/objects/netpod.dm:148]: good idea

**Pull request merged**